### PR TITLE
Upload foreground notification wording

### PR DESCRIPTION
**Pull request by mzorz**
Brings the notification wording and L&F as per the designs in #6735 and discussed in https://github.com/wordpress-mobile/WordPress-Android/pull/6713#issuecomment-337594615 :

screen shot 2017-10-18 at 6 23 26 pm

Unique post case (majority of cases):
- `1 post, 3 of 12 media files remaining`

Unique post case, no media:
- `1 post`

Mixed case (pages and posts):
- `2 posts/pages, 3 of 12 media files remaining`
Note that for this case I decided to keep the `posts/pages` abbreviation instead of showing `1 post, 1 page` format as it takes too much of the screen width and would make the full description not read clearly.

Media only:
- `3 of 12 media files remaining`

Also note the Title reads "Uploading..." instead of using the full site domain for the same reasons noted above

cc @nbradbury

### CODE REVIEW

**Comment by nbradbury** [comment]
The wording is **much** better. 

I ran into an unusual situation, though. I uploaded a post with four images and a page with only text, but the page failed to upload yet the notification continued to say "2 posts/pages." If the page failed, shouldn't the notification have changed to "1 post?"

screenshot_1508612169

**Comment by mzorz** [comment]
>  I uploaded a post with four images and a page with only text, but the page failed to upload yet the notification continued to say "2 posts/pages." If the page failed, shouldn't the notification have changed to "1 post?"

**tl;dr** it counts all items being processed, independently of these having ended uploading successfully.

Fair question though! it is the case that the foreground notification reflects what's happening in the Upload queue: that is, it continues to try whatever is enqueued, and each of the items will have their corresponding success / error notification separately (but this doesn't prevent the foreground notification from going on, and the service to continue to try uploading the other items enqueued for upload).

**Comment by nbradbury** [comment]
If you upload two pages, when one of them completes the wording changes to "1 posts/pages" instead of simply "1 page."

screenshot_1508613021

**Comment by mzorz** [comment]
> If you upload two pages, when one of them completes the wording changes to "1 posts/pages" instead of simply "1 page."

addressed in f6bdd36

**Comment by nbradbury** [comment]
:shipit:
